### PR TITLE
Basic UI

### PR DESCRIPTION
**Pull request by tonnymuchui**
- Create an app.rb :heavy_check_mark: 
- Define the entry point, `main.rb` :heavy_check_mark:

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+  def create_teacher
+    puts 'create a new teacher'
+    puts 'Age:'
+    age = gets.chomp.to_i
+    puts 'Name:'
+    name = gets.chomp
+    puts 'Specialization:'
+    specialization = gets.chomp
+    teacher = Teacher.new(age, name, specialization)
```
Comment (mikethreels) [inline app.rb]: - [ ] when creating a new teacher you are sending the age, name and specialization as arguments in that order, however, if you go to the teacher class you can see that it takes the argument in a different order therefor the application, at the moment, is saving the specialization as the name of the teacher. to solve this you can just switch the arguments you sent like this:
```
teacher = Teacher.new(age, specialization, name)
```

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+    puts 'select person from the list by its number'
+    tem_person = @people.select(&:can_use_services?)
+    tem_person.each_with_index do |person, index|
+      puts "#{index} [#{person.class.name}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+    person_id = gets.chomp.to_i
+    unless (0...tem_person.length).include?(person_id)
+      puts 'sorry wrong input'
+      return
```
Comment (mikethreels) [inline app.rb]: - [ ] please do not exclude the users who have the can_use_services? set to no. as you can see in [the UI example ](https://www.youtube.com/watch?v=vkkgrhD6aXQ&ab_channel=EmiliaAndrzejewska)provided by Microverse these users should be included in the listing when creating a rental as well.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Tonny, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
## Hi @tonnymuchui, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- Correct workflow was used ✔️
- Good commit history ✔️

approval

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**